### PR TITLE
#2328 sentence casing

### PR DESCRIPTION
**Pull request by muryno**
#2328

### CODE REVIEW

**Comment by muryno** [comment]
I was going to ask the root level also 😃.. will update them

**Comment by kylecorry31** [comment]
Can you also confirm that "Title Case" is the correct casing to use on the settings? I would refer to the material design guidelines - and can you post here what you find on that?

Looking at the settings app on my Pixel phone, it looks like sentence case is used rather than title case.

**Comment by muryno** [comment]
I will look at this tomorrow..

**Comment by muryno** [comment]
Sentence casing win😃.  I will update them to sentence casing instead of title casing 


Screenshot 2024-04-20 at 10 59 29
ref: https://m3.material.io/foundations/content-design/style-guide/ux-writing-best-practices

**Comment by kylecorry31** [comment]
Thanks, yes let's go with sentence casing

**Comment by kylecorry31** [comment]
Also for reference, the following should remain in Title Case (product / branded names):
- All tool names
- Weather Monitor
- Backtrack
- Trail Sense

**Comment by muryno** [comment]
I only left Trail Sense

but following the guide to changing 

- All tool names
- Weather Monitor

to sentence casing ... I leave any camelCase the way they are

**Comment by kylecorry31** [comment]
> I only left Trail Sense
> 
> but following the guide to changing
> 
> * All tool names
> * Weather Monitor
> 
> to sentence casing ... I am leaving any camelCase the way they are

Hi, I just looked at the changes and it appears the tool names were changed. I do still want the following to remain in title case (regardless of the material design style guide recommendation):
- All tool names
- Weather Monitor
- Backtrack
- Trail Sense

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -811,8 +811,8 @@
     <string name="no_packing_lists">No packing lists</string>
     <string name="new_packing_list">New packing list</string>
     <string name="delete_pack">Delete pack?</string>
-    <string name="coordinate_format_usng">USNG</string>
-    <string name="coordinate_format_osgb">OSGB</string>
+    <string name="coordinate_format_usng">Usng</string>
```
Comment (kylecorry31) [inline app/src/main/res/values/strings.xml:814]: These should remain all caps since they are acronyms

**Comment by muryno** [comment]
do you want everything in tool quick actions to be title casing or just the title ( All Tool Names)

**Comment by muryno** [comment]
For _Weather Monitor_  do you mean to convert all the  title in weather settings  to title casing

**Comment by kylecorry31** [comment]
> @kylecorry31 For _Weather Monitor_ do you mean to convert all the titles in weather settings to title casing

No, just the "Weather Monitor" string (looks like it only appears on the Weather tool itself)

**Comment by muryno** [comment]
That is now updated.. what of _Monitor Weather_

**Comment by kylecorry31** [comment]
> That is now updated.. what of _Monitor Weather_

Monitor weather can be sentence cased

(Weather Monitor is the branded name of the feature, similar to the tool names - haven't made up my mind if I want to sentence case those, so would prefer not to make that change yet)

File: app/src/main/java/com/kylecorry/trail_sense/tools/tools/ui/Tools.kt [hunk 2]
```diff
@@ -539,7 +539,7 @@ object Tools {
             .flatMap { it.quickActions }
             .distinctBy { it.id }
             .sortedBy { it.id }
-            .map { it.copy(name = it.name.capitalizeWords()) }
+            .map { it.copy(name = it.name) }
```
Comment (kylecorry31) [inline app/src/main/java/com/kylecorry/trail_sense/tools/tools/ui/Tools.kt:542]: Can you undo this change? I don't want the tool quick action names to be changed at this point in time

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1295,7 +1295,7 @@
     <string name="pin">Pin</string>
     <string name="tool_long_press_hint_toast">Long press a tool to see more options</string>
     <string name="pref_tool_sort" translatable="false">pref_tool_sort</string>
-    <string name="tool_quick_actions">Tool quick actions</string>
+    <string name="tool_quick_actions">Tool Quick Actions</string>
```
Comment (kylecorry31) [inline app/src/main/res/values/strings.xml:1298]: This can be sentence cased for now

**Comment by kylecorry31** [comment]
It looks like there are still some changes in progress, let me know when it is ready to review

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -1287,15 +1287,15 @@
     <string name="direction_not_set">Direction not set</string>
     <!--The compass direction / bearing-->
     <string name="direction">Direction</string>
-    <string name="augmented_reality">Augmented reality</string>
+    <string name="augmented_reality">Augmented Reality</string>
     <string name="augmented_reality_description">View nearby beacons and the sun / moon with your camera</string>
     <string name="magnetic_north">Magnetic North</string>
```
Comment (kylecorry31) [inline app/src/main/res/values/strings.xml]: Can you update the magnetic_north string to be sentenced cased as well, to match true_north?

**Comment by muryno** [comment]
> It looks like there are still some changes in progress, let me know when it is ready to review

Actually I only focused on settings part and the root and not all the strings.... you could review and specify which you want to be title case/sentence case

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -161,12 +161,12 @@
...
-    <string name="coordinate_format_decimal_degrees">Decimal Degrees</string>
+    <string name="coordinate_format_decimal_degrees">Decimal degrees</string>
     <!--A coordinate format as such:  X° X.XXX' N-->
-    <string name="coordinate_format_degrees_decimal_minutes">Degrees Decimal Minutes</string>
-    <string name="coordinate_format_degrees_minutes_seconds">Degrees Minutes Seconds</string>
-    <string name="coordinate_format_utm">UTM</string>
-    <string name="coordinate_format_mgrs">MGRS</string>
+    <string name="coordinate_format_degrees_decimal_minutes">Degrees decimal minutes</string>
+    <string name="coordinate_format_degrees_minutes_seconds">Degrees minutes seconds</string>
+    <string name="coordinate_format_utm">Utm</string>
```
Comment (kylecorry31) [inline app/src/main/res/values/strings.xml:168]: Can you make UTM and MGRS all-caps again?

File: app/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -246,7 +246,7 @@
     <string name="pressure_12_hours">12 hours</string>
     <string name="pressure_4_hours">4 hours</string>
     <string name="pref_monitor_weather_summary">In the background</string>
-    <string name="pref_monitor_weather_title">Monitor weather</string>
+    <string name="pref_monitor_weather_title">Monitor Weather</string>
```
Comment (kylecorry31) [inline app/src/main/res/values/strings.xml:249]: This can remain sentence cased

File: app/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -338,7 +338,7 @@
     <string name="pref_max_beacon_distance" translatable="false">pref_max_beacon_distance</string>
     <string name="error_occurred">An error occurred</string>
     <string name="error_occurred_message">Trail Sense ran into an error. Please email the developer to get this issue fixed.</string>
-    <string name="tool_user_guide_title">User Guide</string>
+    <string name="tool_user_guide_title">User guide</string>
```
Comment (kylecorry31) [inline app/src/main/res/values/strings.xml:341]: This should remain Title Cased

File: app/src/main/res/values/strings.xml [hunk 8]
```diff
@@ -361,7 +361,7 @@
     <string name="category_safety">Safety</string>
     <string name="category_natural">Natural</string>
     <string name="toast_destination_bearing_set">Destination bearing set</string>
-    <string name="tool_cliff_height_title">Cliff Height</string>
+    <string name="tool_cliff_height_title">Cliff height</string>
```
Comment (kylecorry31) [inline app/src/main/res/values/strings.xml:364]: This should remain Title Cased

File: app/src/main/res/values/strings.xml [hunk 9]
```diff
@@ -401,7 +401,7 @@
     <string name="thunder">Thunder</string>
     <string name="reset">Reset</string>
     <string name="tool_lightning_description">Measure the distance from a lightning strike</string>
-    <string name="tool_lightning_title">Lightning Strike Distance</string>
+    <string name="tool_lightning_title">Lightning strike distance</string>
```
Comment (kylecorry31) [inline app/src/main/res/values/strings.xml:404]: This should remain Title Cased

File: app/src/main/res/values/strings.xml [hunk 10]
```diff
@@ -435,7 +435,7 @@
     <string name="clock_sync_time_settings">Open time settings?</string>
     <string name="magnetic_field_format_precise">%s uT</string>
     <string name="threshold">Threshold</string>
-    <string name="tool_metal_detector_title">Metal Detector</string>
+    <string name="tool_metal_detector_title">Metal detector</string>
```
Comment (kylecorry31) [inline app/src/main/res/values/strings.xml:438]: This should remain Title Cased

File: app/src/main/res/values/strings.xml [hunk 11]
```diff
@@ -451,7 +451,7 @@
     <string name="unit_inches">Inches</string>
     <string name="map_distance">Map: %s</string>
     <string name="map_fractional_separator">:</string>
-    <string name="tool_white_noise_title">White Noise</string>
+    <string name="tool_white_noise_title">White noise</string>
```
Comment (kylecorry31) [inline app/src/main/res/values/strings.xml:454]: This should remain Title Cased

File: app/src/main/res/values/strings.xml [hunk 12]
```diff
@@ -1380,7 +1380,7 @@
     <!--The camera projection type for augmented reality (ex. perspective)-->
     <string name="projection_type">Augmented reality projection</string>
     <!--Camera projection using intrinsics matrix-->
-    <string name="projection_estimated_intrinsics">Estimated Intrinsics</string>
+    <string name="projection_estimated_intrinsics">Estimated intrinsics</string>
     <!--Camera projection using intrinsics matrix-->
     <string name="projection_manufacturer_intrinsics">Manufacturer Intrinsics</string>
```
Comment (kylecorry31) [inline app/src/main/res/values/strings.xml:1385]: This should be sentence cased to match the estimated intrinsics

**Comment by kylecorry31** [comment]
> > It looks like there are still some changes in progress, let me know when it is ready to review
> 
> Actually I only focused on settings part and the root and not all the strings.... you could review and specify which you want to be title case/sentence case ... as I am getting confuse since it is not consistence ..

Done, I left some comments - the scope of this was just the settings for now, so that is fine.

Sorry about the confusion - this issue was not ready for community contribution, so I didn't have all the details ready (I was going to make it open for contribution, so no worries about picking it up early).

File: app/src/main/res/values/strings.xml (hunk 12 above)
Comment (muryno) [reply app/src/main/res/values/strings.xml:1385]: you mean title case? as it is already sentence case

**Comment by muryno** [comment]
all updated

**Comment by kylecorry31** [comment]
Looks good, thanks!
